alfresco-realm.json

```diff
@@ -494,7 +494,7 @@
       "enabled": true,
       "clientAuthenticatorType": "client-secret",
       "secret": "**********",
-      "redirectUris": [],
+      "redirectUris": [ "http://localhost:*" ],
```

Comment:
Perhaps this should be a more obvious placeholder, something like `alfrescoClientRedirectUris` rather than `http://localhost:*`?

Reply:
👍